Mixpanel/MPNotificationViewController.m

```diff
@@ -269,6 +287,13 @@ - (void)viewDidLoad
     [self.view addGestureRecognizer:pan];
 }
 
+- (void)initializeMiniNotification {
+    self.backgroundColor = [UIColor blackColor];
+    self.view.backgroundColor = self.backgroundColor;
+    UIColor *backgroundColorWithAlphaComponent = [self.backgroundColor colorWithAlphaComponent:0.901f];
+    self.view.backgroundColor = backgroundColorWithAlphaComponent;
```

Comment:
What is going on here?

You set `self.view.backgroundColor` twice.

You could just set it to `[UIColor colorWithWhite:0 alpha:0.901f]`